apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.html

```diff
@@ -153,7 +152,8 @@ <h3 class="font-16 d-flex" [checkPermission]="['MENU', 'MANAGE']" dropdownToggle
             </div>
             <div class="menu-item-wrapper font-14">
                 <ul class="list-style-none giddh-primary-items">
-                    <li class="main-item btn-group" *ngFor="let menu of allItems; trackBy: trackItems" dropdown
+
+                    <li class="main-item btn-group"  (click)="addClass = !addClass" *ngFor="let menu of allItems; trackBy: trackItems" dropdown
```

Comment:
Remove  `(click)="addClass = !addClass"` if it is unused

Reply:
Done